cocotb/share/makefiles/Makefile.inc

```diff
@@ -55,7 +55,13 @@ endif
 export OS
 
 # this ensures we use the same python as the one cocotb was installed into
-PYTHON_BIN ?= $(shell cocotb-config --python-bin)
+# realpath to convert windows paths to unix paths, like cygpath -u
+PYTHON_BIN ?= $(realpath $(shell cocotb-config --python-bin))
+
+# Set PYTHONHOME to properly populate sys.path in embedded python interpreter
+export PYTHONHOME := $(shell $(PYTHON_BIN) -c 'from distutils.sysconfig import get_config_var; print(get_config_var("prefix"))')
+
+LIB_DIR=$(COCOTB_PY_DIR)/cocotb/libs
```

Comment:
Is there a reason this happens before the other includes?

Reply:
Lagecy